eth/transactionManager.go

```diff
@@ -0,0 +1,259 @@
+	tm.cond.L.Unlock()
+	tm.cond.Signal()
+
+	glog.Infof("\n%vEth Transaction%v\n\nInvoking transaction: \"%v\". Inputs: \"%v\"  Hash: \"%v\". \n\n%v\n", strings.Repeat("*", 30), strings.Repeat("*", 30), txLog.method, txLog.inputs, tx.Hash().String(), strings.Repeat("*", 75))
+
+	return nil
+}
+
+func (tm *TransactionManager) Subscribe(sink chan<- *transactionReceipt) event.Subscription {
+	return tm.scope.Track(tm.feed.Subscribe(sink))
+}
+
+func (tm *TransactionManager) Start() {
+	tm.checkTxLoop()
+}
+
+func (tm *TransactionManager) Stop() {
+	tm.scope.Close()
+	close(tm.quit)
+}
+
+func (tm *TransactionManager) wait(tx *types.Transaction) (*types.Receipt, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), tm.txTimeout)
+	defer cancel()
+
+	receipt, err := bind.WaitMined(ctx, tm.eth, tx)
+	if err != nil {
+		return nil, err
+	}
+
+	return receipt, nil
+}
+
+func (tm *TransactionManager) replace(tx *types.Transaction) (*types.Transaction, error) {
+	_, pending, err := tm.eth.TransactionByHash(context.Background(), tx.Hash())
+	// Only return here if the error is not related to the tx not being found
+	// Presumably the provided tx was already broadcasted at some point, so even if for some reason the
+	// node being used cannot find it, the originally broadcasted tx is still valid and might be sitting somewhere
+	if err != nil && err != ethereum.NotFound {
+		return nil, err
+	}
+	// If tx was found
+	// If `pending` is false, the tx was mined and included in a block
+	if err == nil && !pending {
+		return nil, ErrReplacingMinedTx
+	}
+
+	// Updated gas price must be at least 10% greater than the gas price used for the original transaction in order
+	// to submit a replacement transaction with the same nonce. 10% is not defined by the protocol, but is the default required price bump
+	// used by many clients: https://github.com/ethereum/go-ethereum/blob/01a7e267dc6d7bbef94882542bbd01bd712f5548/core/tx_pool.go#L148
+	// We add a little extra in addition to the 10% price bump just to be sure
+	gasPrice := calcReplacementGasPrice(tx)
+
+	suggestedGasPrice := tm.gpm.GasPrice()
+
+	// If the suggested gas price is higher than the bumped gas price, use the suggested gas price
+	// This is to account for any wild market gas price increases between the time of the original tx submission and time
+	// of replacement tx submission
+	// Note: If the suggested gas price is lower than the bumped gas price because market gas prices have dropped
+	// since the time of the original tx submission we cannot use the lower suggested gas price and we still need to use
+	// the bumped gas price in order to properly replace a still pending tx
+	if suggestedGasPrice.Cmp(gasPrice) == 1 {
+		gasPrice = suggestedGasPrice
+	}
+
+	// Bump gas price exceeds max gas price, return early
+	max := tm.gpm.MaxGasPrice()
+	if gasPrice.Cmp(max) > 0 {
+		return nil, fmt.Errorf("Replacement gas price exceeds max gas price suggested=%v max=%v", gasPrice, max)
```

Comment:
```suggestion
		return nil, fmt.Errorf("replacement gas price exceeds max gas price suggested=%v max=%v", gasPrice, max)
```

To follow convention of error messages in lowercase (except if its extra annotation text in an error log statement)